controllers/storagecluster/external_resources.go

```diff
@@ -291,6 +301,41 @@ func (obj *ocsExternalResources) ensureCreated(r *StorageClusterReconciler, inst
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageClusterReconciler) checkSCsAndCallCreateDefaultClaim(instance *ocsv1.StorageCluster) error {
+	cephFsStorageClassName := generateNameForCephFilesystemSC(instance)
+	cephRbdStorageClassName := generateNameForCephBlockPoolSC(instance)
+
+	cephFsStorageClass := &storagev1.StorageClass{}
+	if err := r.Client.Get(r.ctx, types.NamespacedName{Name: cephFsStorageClassName, Namespace: instance.Namespace}, cephFsStorageClass); err != nil {
+		return fmt.Errorf("no storageClass found with name %s. %v", cephFsStorageClassName, err)
+	}
+
+	cephRbdStorageClass := &storagev1.StorageClass{}
+	if err := r.Client.Get(r.ctx, types.NamespacedName{Name: cephRbdStorageClassName, Namespace: instance.Namespace}, cephRbdStorageClass); err != nil {
```

Comment:
storageclass is not a namespace scoped object. remove Namespace

Reply:
ah right, removed